### ISSUE TITLE
Stored Procedure and Function Schemas missing in Reverse Engineer starting with version 2.5.717 installed on 7/24/2021

### ISSUE THREAD

**Issue by bhroberts52** [ErikEJ/EFCorePowerTools#1056]
As of version 2.5.717 the dialog for reverse engineering a SQL database is not picking up schemas other than dbo when looking at Stored Procedures and Functions.  (Tables and Views are okay.)
[CreateDatabaseScript.txt](https://github.com/ErikEJ/EFCorePowerTools/files/6874256/CreateDatabaseScript.txt)

### Steps to reproduce
Try doing a reverse engineer on a sql database that has more than the dbo schema represented in Stored Procedures and/or Functions

### Further technical details
EF Core Power Tools version: 2.5.717

Database engine: SQL Server, Azure SQL

Visual Studio version: Visual Studio 2019 16.9.1  and Visual Studio 2019 16.11.0 preview 3.0

**Comment by ErikEJ** [comment]
Thanks, I will investigate tomorrow.

**Comment by ErikEJ** [comment]
I attempted a fix in the latest daily build (718 or later) - would be grateful if you could confirm.

**Comment by bhroberts52** [comment]
Hello Mr Jensen,

 

First, thanks again for this wonderful tool.

 

I downloaded build 718 and found the following:

1.	Visual Studio 16.11.0 preview 3.0 crashes and closes when I try to open the EF Core Power Tools
2.	In Visual 16.9.1, EF Core Power Tools opens and works as advertised using my local SQL database connection 😊
3.	In Visual  Studio 16.9.1 the connection to the Azure SQL Database would not work initially.  It didn’t crash and close, but indicated it was downloading  but wouldn’t complete.  I deleted the stored Azure SQL connection, and then added it back in, and then it worked.

 

I am back in business, but am unable to use VS 16.11.0 preview 3.0 (which is no big deal for me).

 

Thanks again

 

Burton Roberts

Extended Day Services

(412) 608-4205

**Comment by ErikEJ** [comment]
Thanks for the feedback. If you like my free tools I would be grateful for a review on VS Marketplace.

**Comment by bhroberts52** [comment]
Will do!

 

Burton Roberts

Extended Day Services

(412) 608-4205

**Comment by bhroberts52** [comment]
After reinstalling Visual Studio 16.11.0 preview 3.0, everything seems to be working OK!

 

Thanks again.  I wrote a review.

 

Burton Roberts

Extended Day Services

(412) 608-4205

 

From: ***@***.*** ***@***.***> 
Sent: Monday, July 26, 2021 8:54 AM
To: 'ErikEJ/EFCorePowerTools' ***@***.***>
Subject: RE: [ErikEJ/EFCorePowerTools] Stored Procedure and Function Schemas missing in Reverse Engineer starting with version 2.5.717 installed on 7/24/2021 (#1056)

 

Hello Mr Jensen,

 

First, thanks again for this wonderful tool.

 

I downloaded build 718 and found the following:

1.	Visual Studio 16.11.0 preview 3.0 crashes and closes when I try to open the EF Core Power Tools
2.	In Visual 16.9.1, EF Core Power Tools opens and works as advertised using my local SQL database connection 😊
3.	In Visual  Studio 16.9.1 the connection to the Azure SQL Database would not work initially.  It didn’t crash and close, but indicated it was downloading  but wouldn’t complete.  I deleted the stored Azure SQL connection, and then added it back in, and then it worked.

 

I am back in business, but am unable to use VS 16.11.0 preview 3.0 (which is no big deal for me).

 

Thanks again

 

Burton Roberts

Extended Day Services

(412) 608-4205
